<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrarPrestamo</class>
 <widget class="QDialog" name="RegistrarPrestamo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="CbxLaboratorio">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Laboratorio</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Dni:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LnDni">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
  </widget>
  <widget class="QComboBox" name="CbxEquipo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Equipo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbRegistrar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LnPago">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Pago:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dtDevolucion">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dtPrestamo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha Devolucion:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha Prestamo</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>CbxLaboratorio</tabstop>
  <tabstop>CbxEquipo</tabstop>
  <tabstop>LnDni</tabstop>
  <tabstop>LnPago</tabstop>
  <tabstop>PbRegistrar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
